src/Rector/Contrib/PHPUnit/SpecificMethodCountRector.php

```diff
@@ -60,15 +77,21 @@ public function isCandidate(Node $node): bool
             return false;
         }
 
-        return $secondArgumentValue->name->toString() === 'count';
+        $coutableMethod = $secondArgumentValue->name->toString();
+
+        return in_array($coutableMethod, [
+            'count', 'sizeof',
+        ]);
     }
 
     /**
      * @param MethodCall $methodCallNode
      */
     public function refactor(Node $methodCallNode): ?Node
     {
-        $methodCallNode->name = new Identifier('assertCount');
+        $oldMethodName = $methodCallNode->name->toString();
+
+        $methodCallNode->name = new Identifier($this->renameMethodsMap[$oldMethodName]);
```

Comment:
@TomasVotruba I handled it by calling the key in `$this->renameMethodsMap`, since we already filtered the assertions methods in [`L53`](https://github.com/rectorphp/rector/pull/198/files#diff-a7ca84c531c86cadc201c1d5354dd0baR58). Is it right? :confused:

Reply:
Nice one! :+1:

Reply:
Sometimes I forget this `PHP` methods :joy: